dotCMS/src/main/webapp/html/js/dotcms/dijit/form/ContentSelector.jsp

```diff
@@ -1,5 +1,5 @@
 <div id="${id}">
-	<div dojoAttachPoint="dialog" dojoType="dijit.Dialog" dojoAttachEvent='onHide:_clearSearch' class="related-content-dialog">
+	<div dojoAttachPoint="dialog" dojoType="dijit.Dialog" dojoAttachEvent='onHide:_clearSearch' class="related-content-dialog" style="width:90%;height:90%">

```

Comment:
The style changes should be inside the `related-content-dialog` class

Reply:
done